src/components/BookItem/BookItem.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from './_BookItem.scss';
```

Comment:
I think this can just be
`import './_BookItem.scss`;

I notice that it works with out naming it styles anyway. I think we will talk about this in tonight's stand up. I'm not totally clear on all the different ways to access css/scss files!

Reply:
done